spring/pom.xml

```diff
@@ -71,10 +75,45 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
```

Comment:
why do we need this now? Does that affect the non-native build?

Reply:
This was the attempt to get Spring AOT processing (which is only part of Boot) working. But, it did not end up working anyway, and we skip building the test jar. So I'll remove this for now.